misc/ls-extensions/modules/json-to-record-converter/src/main/java/io/ballerina/converters/JsonToRecordConverter.java

```diff
@@ -135,6 +141,30 @@ public static JsonToRecordResponse convert(String jsonString, String recordName,
         return response;
     }
 
+    /**
+     * This method takes in a json string and returns the Ballerina code block.
+     *
+     * @param jsonString Json string for the schema
+     * @param recordName Name of the generated record
+     * @param isRecordTypeDesc To denote final record, a record type descriptor
+     * @param isClosed To denote the whether the response record is closed
+     * @param forceFormatRecordTypeDesc To denote whether the inline records to be formatted for multi-line or in-line
+     * @return {@link String} Ballerina code block
+     * @throws IOException In case of Json parse error
+     * @throws JsonToRecordConverterException In case of invalid schema
+     * @throws FormatterException In case of invalid syntax
+     */
+    public static JsonToRecordResponse convert(String jsonString, String recordName, boolean isRecordTypeDesc,
+                                               boolean isClosed, boolean forceFormatRecordTypeDesc) throws IOException,
+            JsonToRecordConverterException, FormatterException {
+        JsonToRecordResponse response = convert(jsonString, recordName, isRecordTypeDesc, isClosed);
+        FormattingOptions formattingOptions = FormattingOptions.builder()
+                .setForceFormattingOptions(ForceFormattingOptions.builder()
+                        .setForceFormatRecordFields(forceFormatRecordTypeDesc).build()).build();
+        response.setCodeBlock(Formatter.format(response.getCodeBlock(), formattingOptions));
+        return response;
+    }
```

Comment:
Let's do the overloading the other way around. We can use this method inside the deprecated method instead right?